lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -61,11 +61,18 @@ sub register {
         });
 
     $app->helper(
+        bug_report_actions => sub {
+            my ($c) = @_;
+            return $c->include_branding('external_reporting');
```

Comment:
I think you need a stub in the plain branding - or you will break everybody else's instance